e2e/adapters-e2e/src/components/router-head/router-head.tsx

```diff
@@ -12,13 +12,17 @@ export const RouterHead = component$(() => {
       <link rel="canonical" href={loc.url.href} />
       <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-      {head.meta.map((m) => (
-        <meta key={m.key} {...m} />
-      ))}
+      {head.meta.map((m, idx) => {
+        const { key, ...rest } = m;
+        const metaKey = key ?? m.name ?? m.property ?? `meta-${idx}`;
+        return <meta key={metaKey} {...rest} />;
```

Comment:
Actually, I think this isn't necessary. The key is only used to find DOM elements again in a reordered array, but normally the order stays the same or most of the elements will be replaced.

Probably this code can just be `{head.meta.map(m => <meta {...m} />}`